homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -31,10 +31,9 @@ def __init__(self, discovery_info):
         """Initialize the LG speakers."""
         self._host = discovery_info.get("host")
         self._port = discovery_info.get("port")
-        properties = discovery_info.get("properties")
-        self._uuid = properties.get("UUID")
+        self._hostname = discovery_info.get("hostname")
 
-        self._name = ""
+        self._name = self._hostname.split(".")[0]
```

Comment:
`self._hostname` can be None.

Reply:
Isn't the hostname/domain a necessary/required field in DNS-SD?

I only added this to have a proper entity_id since if i understand it correctly the name is used as entity_id (cleaned). The "real" name/label is later set/updated by requesting it from the soundbar API.

Reply:
If it's required, then don't use `.get()` to get the value from `discovery_info`. Instead

Reply:
If this is best practice should we do the same for `host` and `port`?

for now i am testing the `hostname` in square brackets change.

Reply:
Correct. If they are always there, use square brackets.